app/tests/spec/models/mixins/resume-token.js

```diff
@@ -10,23 +10,32 @@ define(function (require, exports, module) {
   var Cocktail = require('cocktail');
   var ResumeToken = require('models/resume-token');
   var ResumeTokenMixin = require('models/mixins/resume-token');
+  var vat = require('lib/vat');
 
   var assert = chai.assert;
 
   describe('models/mixins/resume-token', function () {
     var model;
-    var CAMPAIGN = 'campaign id';
-    var RESUME_DATA = {
+    var CAMPAIGN = 'deadbeef';
+    var RESUME_SCHEMA = {
+      campaign: vat.hex().len(8)
```

Comment:
Have called it out explicitly in https://github.com/mozilla/fxa-content-server/issues/2920.